lighthouse-core/test/chromium-web-tests/test-locally.sh

```diff
@@ -13,10 +13,10 @@ LH_ROOT="$SCRIPT_DIR/../../.."
 TEST_DIR="$LH_ROOT/.tmp/chromium-web-tests"
 
 # Setup dependencies.
-export DEPOT_TOOLS="$TEST_DIR/depot-tools"
```

Comment:
doh! thought I fixed that.

Reply:
i kinda remember seeing that change too, but there's no sign of it.

there were a few comments i made that got hidden as there were commits pushed whilst i was making the review.